src/components/Navbar.tsx

```diff
@@ -73,7 +72,7 @@ export default function Navbar() {
                           alt=""
                           className="h-8 w-8 rounded-full"
                           height="32"
-                          src={user.imageUrl}
+                          src={user?.image as string}
```

Comment:
I think we can remove the `as` here and all other places after addressing the issue above

Reply:
See my other comment to understand the issue.